org.bluej.BlueJ.json

```diff
@@ -0,0 +1,73 @@
+{
+  "app-id" : "org.bluej.BlueJ",
+  "branch" : "stable",
+  "runtime" : "org.gnome.Platform",
```

Comment:
Doesn't BlueJ work with the Freedesktop SDK/Platform, since they already contain GTK?

Reply:
It kinda does but it uses a native file picker, so if you use a GTK theme other than Adwaita:light, the file picker doesn't look right on freedesktop platform.

Reply:
Ah, alright